community-plugins.json

```diff
@@ -7577,7 +7577,7 @@
         "id": "image-converter",
         "name": "Image Converter",
         "author": "xRyul",
-        "description": "Convert, compress and resize images from one format to another by dragging and dropping or pasting into the note.",
+        "description": "Convert, compress, resize, annotate, markup, draw, crop, rotate, flip, align images directly in Obsidian. Drag-resize, rename with variables, batch process. WEBP, JPG, PNG, HEIC, TIF.",
```

Comment:
Don't include the word "Obsidian" in the description.
It should be pretty obvious that this is for Obsidian.